uaclient/cli.py

```diff
@@ -876,6 +886,37 @@ def action_disable(args, *, cfg, **kwargs):
     return 0 if ret else 1
 
 
+def _enable(
```

Comment:
Just a nitpick here, should we move this function to the actions module ?

Reply:
Hmmm, yes probably, I'll go ahead and do that here